docs/src/user-docs/guides-quick-start.md

```diff
@@ -64,6 +66,19 @@ The integer parameters and return value are `Serializable` values.
 The `SPIDER_REGISTER_TASK` macro at the bottom of `src/tasks.cpp` is how we inform Spider that a
 function should be treated as a task.
 
+# (Optional) Installing dependencies
+
+You can install `Spider` dependencies locally using the `task` command by running:
+
+```shell
+task deps:lib_install
+```
+
+This will install all dependencies in the `build/deps` directory.
+
+Alternatively, you can install the dependencies yourself. Check the `CMakeLists.txt` file for the
+list of dependencies.
```

Comment:
```suggestion
Alternatively, you can install the dependencies to the system. See the `install-all-run` task in
[deps-task.yaml] for the list of dependencies to install.
```

Add a link to the file on GitHub (we can't link to it locally since Sphinx will turn it into a download link).